src/azure-cli/azure/cli/command_modules/appconfig/_help.py

```diff
@@ -590,3 +592,66 @@
           text:
             az appconfig feature filter list --endpoint https://myappconfiguration.azconfig.io --feature color --all --auth-mode login
     """
+
+
+helps['appconfig snapshot'] = """
+    type: group
+    short-summary: Manage snapshots associated with an app configuration store.
+    """
+
+helps['appconfig snapshot create'] = """
+    type: command
+    short-summary: Create an app configuration snapshot.
+    examples:
+        - name: Create a snapshot MySnapshot of all keys starting with 'Test' in configuration store MyAppConfiguration.
+          text:
+            az appconfig snapshot create -s MySnapshot -n MyAppConfiguration --filters '{\"key\":\"Test*\"}'
+        - name: Create a snapshot MySnapshot of all keys starting with 'abc' and a retention period of 5 minutes.
+          text:
+            az appconfig snapshot create -s MySnapshot -n MyAppConfiguration --filters '{\"key\":\"abc*\"}' --retention-period 300
+        - name: Create a snapshot of all keys starting with 'app/' with the label 'dev' as default, and override the key-values with keys with the label 'prod' if they exist.
```

Comment:
I think the more common example, is the first filter having no label, and then the second filter overriding with "prod"

Reply:
This has been updated.